### PR TITLE
Explicitly setting cell timestamp

### PR DESCRIPTION
**Pull request by malish8632**
Fixing cell timestamp to be used by BigTable built-in garbage collection.

### CODE REVIEW

**Comment by hexedpackets** [comment]
I guess the timestamp column already exists and we just weren't using it, so should the amount of bytes stored stay the same? Or will this increase the size of the rows in BigTable?

File: metric/bigtable/src/main/java/com/spotify/heroic/metric/bigtable/api/Mutations.java [hunk 1]
```diff
@@ -66,6 +68,7 @@ public Builder setCell(
                     .newBuilder()
                     .setFamilyName(family)
                     .setColumnQualifier(columnQualifier)
+                    .setTimestampMicros(TimeUnit.MILLISECONDS.toMicros(DateTime.now().getMillis()))
```
Comment (dmichel1) [inline metric/bigtable/src/main/java/com/spotify/heroic/metric/bigtable/api/Mutations.java]: Could also set this to `-1` to use the bigtable server time - https://cloud.google.com/bigtable/docs/reference/data/rpc/google.bigtable.v2#google.bigtable.v2.Mutation.SetCell

Comment (malish8632) [reply metric/bigtable/src/main/java/com/spotify/heroic/metric/bigtable/api/Mutations.java]: good catch. changed

**Comment by malish8632** [comment]
> I guess the timestamp column already exists and we just weren't using it, so should the amount of bytes stored stay the same? Or will this increase the size of the rows in BigTable?

I'm going to follow up on this one with Bigtable gurus. It is probably will increase the size but we need it to use built-in garbage collection.

**Pull request merged**